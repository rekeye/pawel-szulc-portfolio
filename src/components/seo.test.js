import React from "react";
import renderer from "react-test-renderer";
import { useStaticQuery } from "gatsby";

import Helmet from "react-helmet";
import SEO from "./seo";

describe("SEO component", () => {
  beforeAll(() => {
    useStaticQuery.mockReturnValue({
      site: {
        siteMetadata: {
          title: "PaweÅ‚ Szulc",
          description:
            "Quis velit esse quis sit nulla. Reprehenderit amet enim reprehenderit voluptate quis officia commodo ex ad.",
        },
      },
    });
  });

  it("renders the tests correctly", () => {
    const tree = renderer.create(<SEO title='Projekty' />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
