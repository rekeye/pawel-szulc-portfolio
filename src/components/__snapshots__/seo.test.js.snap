// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO component renders the tests correctly 1`] = `null`;
